src/main/scala/io/iohk/ethereum/jsonrpc/http/JsonRpcHttpServer.scala

```diff
@@ -52,6 +55,10 @@ class JsonRpcHttpServer(jsonRpcController: JsonRpcController, config: JsonRpcHtt
     complete(jsonRpcController.handleRequest(request))
   }
 
+  private def handleBatchRequest(requests: Seq[JsonRpcRequest]) = {
+    complete(Future.sequence(requests.map(request => jsonRpcController.handleRequest(request))))
```

Comment:
I wonder if should synchronise those calls in case the caller expects them to be performed in the given order? IMO, finding this out is outside the scope of this PR.